Dockerfile

```diff
@@ -46,5 +47,5 @@ WORKDIR /home/appuser
 USER appuser
 ENV PYTHONUNBUFFERED=1
 
-ENTRYPOINT ["wps"]
+ENTRYPOINT ["opentelemetry-instrument", "wps"]
```

Comment:
This means that opentelemetry-instrument is started which then starts wps, right?